<?xml version='1.0' encoding='UTF-8'?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>738</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>This wizard will take you through the steps to read in your spectrum from your data file. Once you are done, you have the option to save this configuration as a YAML file which will allow you to read in similar files automatically in future.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item column="0" row="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Dispersion</string>
         </property>
        </widget>
       </item>
       <item colspan="3" column="4" row="0" rowspan="17">
        <layout class="QVBoxLayout" name="layout_preview" />
       </item>
       <item column="0" row="1">
        <widget class="QLabel" name="label_dispersion_dataset">
         <property name="text">
          <string>{{dataset}} the dispersion is stored in:</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="1">
        <widget class="QComboBox" name="combo_dispersion_dataset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="2">
        <widget class="QLabel" name="label_dispersion_component">
         <property name="text">
          <string>Which component is it stored in?</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="2">
        <widget class="QComboBox" name="combo_dispersion_component">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="3">
        <widget class="QLabel" name="label_dispersion_units">
         <property name="text">
          <string>What units is the dispersion in?</string>
         </property>
        </widget>
       </item>
       <item column="1" row="3">
        <widget class="QComboBox" name="combo_dispersion_units">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="2" row="3">
        <widget class="QLineEdit" name="value_dispersion_units">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string />
         </property>
        </widget>
       </item>
       <item column="3" row="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item column="2" row="4">
        <widget class="QLabel" name="label_dispersion_status">
         <property name="text">
          <string>{{status line}}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item column="0" row="5">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Data</string>
         </property>
        </widget>
       </item>
       <item column="0" row="6">
        <widget class="QLabel" name="label_data_dataset">
         <property name="text">
          <string>{{dataset}} the data is stored in</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="6">
        <widget class="QComboBox" name="combo_data_dataset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="7">
        <widget class="QLabel" name="label_data_component">
         <property name="text">
          <string>Which component is it stored in?</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="7">
        <widget class="QComboBox" name="combo_data_component">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="8">
        <widget class="QLabel" name="label_data_units">
         <property name="text">
          <string>What units is the data in?</string>
         </property>
        </widget>
       </item>
       <item column="1" row="8">
        <widget class="QComboBox" name="combo_data_units">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="2" row="8">
        <widget class="QLineEdit" name="value_data_units">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>90</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>90</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="placeholderText">
          <string />
         </property>
        </widget>
       </item>
       <item column="2" row="9">
        <widget class="QLabel" name="label_data_status">
         <property name="text">
          <string>{{status line}}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item column="0" row="10">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Uncertainty</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="10">
        <widget class="QCheckBox" name="bool_uncertainties">
         <property name="text">
          <string>File includes uncertainties</string>
         </property>
        </widget>
       </item>
       <item column="0" row="11">
        <widget class="QLabel" name="label_uncertainty_dataset">
         <property name="text">
          <string>{{dataset}} the uncertainty is stored in</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="11">
        <widget class="QComboBox" name="combo_uncertainty_dataset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="12">
        <widget class="QLabel" name="label_uncertainty_component">
         <property name="text">
          <string>Which component is it stored in?</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="12">
        <widget class="QComboBox" name="combo_uncertainty_component">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="13">
        <widget class="QLabel" name="label_uncertainty_type">
         <property name="text">
          <string>How are the uncertainties stored?</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="13">
        <widget class="QComboBox" name="combo_uncertainty_type">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToMinimumContentsLength</enum>
         </property>
        </widget>
       </item>
       <item column="0" row="15">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Bit Mask</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="15">
        <widget class="QCheckBox" name="bool_mask">
         <property name="text">
          <string>File includes mask</string>
         </property>
        </widget>
       </item>
       <item column="0" row="16">
        <widget class="QLabel" name="label_mask_dataset">
         <property name="text">
          <string>{{dataset}} the mask is stored in</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="16">
        <widget class="QComboBox" name="combo_mask_dataset">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item column="0" row="17">
        <widget class="QLabel" name="label_uncertainty_component">
         <property name="text">
          <string>Which component is it stored in?</string>
         </property>
        </widget>
       </item>
       <item colspan="2" column="1" row="17">
        <widget class="QComboBox" name="combo_mask_component">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item colspan="3" column="4" row="17" rowspan="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>If you would like to read other files like this in future, you can set a name for the loader below and click on Save to YAML (be sure to save it to a file with a *.yaml extension, in the ~/.specviz directory). But this is optional, so you can just click OK and open this in Specviz as a one-off.</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignBottom|Qt::AlignJustify</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      <item column="0" row="18">
        <widget class="QLabel" name="label_data_dataset">
         <property name="font">
          <font>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>Bit mask description file</string>
         </property>
        </widget>
       </item>
       <item column="1" row="18">
        <widget class="QPushButton" name="button_select_mask_file">
         <property name="text">
          <string>Select file</string>
         </property>
        </widget>
       </item>
       <item column="2" row="18">
        <widget class="QLabel" name="label_filename">
         <property name="text">
          <string>{{filename}}</string>
         </property>
        </widget>
       </item>
       <item column="1" row="19">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Bit column</string>
         </property>
        </widget>
       </item>
       <item column="2" row="19">
        <widget class="QComboBox" name="combo_bit_id" />
       </item>
       <item column="1" row="20">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Bit name</string>
         </property>
        </widget>
       </item>
       <item column="2" row="20">
        <widget class="QComboBox" name="combo_bit_name" />
       </item>
       <item column="4" row="20">
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Loader Name</string>
         </property>
        </widget>
       </item>
       <item column="5" row="20">
        <widget class="QLineEdit" name="loader_name">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item column="6" row="20">
        <widget class="QPushButton" name="button_save_yaml">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Save to YAML</string>
         </property>
        </widget>
       </item>
       <item column="1" row="21">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Bit description</string>
         </property>
        </widget>
       </item>
       <item column="2" row="21">
        <widget class="QComboBox" name="combo_bit_description" />
       </item>
       <item column="5" row="21">
        <widget class="QLabel" name="label_loader_name_status">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true" />
         </property>
         <property name="text">
          <string>{{status line}}</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="button_yaml">
       <property name="text">
        <string> Preview YAML</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_ok">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources />
 <connections />
</ui>